applicationcharge_test.go

```diff
@@ -33,7 +33,7 @@ func applicationChargeTests(t *testing.T, charge ApplicationCharge) {
 		},
 		{
 			"ConfirmationURL",
-			"https://apple.myshopify.com/admin/charges/1017262355/confirm_application_charge?sign" +
+			"https://apple.myshopify.com/admin/api/2019-04/charges/1017262355/confirm_application_charge?sign" +
```

Comment:
Next ShopifyAPI version will be `2019-07`. I think here `GetAdminVersionedApiPathPrefix()` function could be used for reducing size of next PR and decreasing number of changes.

Reply:
If you see my comment here:

https://github.com/bold-commerce/go-shopify/pull/48#discussion_r277762885

Which one would you prefer?